### PR TITLE
Jpnlang 0002 user service post api

### PR DESCRIPTION
**Pull request by shoaib812**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,5 +9,7 @@ server.port = 8081
 #logging.level.org.hibernate=ERROR
 #logging.level.net.guides=DEBUG
 
-logging.file=java-challenge
-logging.path=D:\1_Repo\java-challenge
\ No newline at end of file
+server.port = 8082
```
Comment (Mohdzaved2434) [inline src/main/resources/application.properties:11]: Can you remove server.port = 8081 from line 6?

File: src/main/java/com/jpn/lang/controller/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        employeeResponse.setId(employee.getEmployeeId());
+        employeeResponse.setUsername(employee.getUsername());
+        employeeResponse.setMail(employee.getMail());
+        employeeResponse.setDob(employee.getDob());
+        employeeResponse.setAddress(employee.getAddress());
+
+        return employeeResponse;
+    }
+
+
```
Comment (Mohdzaved2434) [inline src/main/java/com/jpn/lang/controller/EmployeeController.java]: Please remove empty line?

File: src/main/java/com/jpn/lang/controller/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
+package com.jpn.lang.controller;
+
+
+import com.jpn.lang.entity.Employee;
+import com.jpn.lang.model.request.EmployeePostRequest;
+import com.jpn.lang.model.response.EmployeeResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```
Comment (Mohdzaved2434) [inline src/main/java/com/jpn/lang/controller/EmployeeController.java]: Next time please try to import fully qualified package name.

File: src/main/resources/application.properties (hunk 1 above)
Comment (shoaib812) [reply src/main/resources/application.properties:11]: I have removed this line.

File: src/main/java/com/jpn/lang/controller/EmployeeController.java (hunk 2 above)
Comment (shoaib812) [reply src/main/java/com/jpn/lang/controller/EmployeeController.java]: I have removed empty lines.

File: src/main/java/com/jpn/lang/controller/EmployeeController.java (hunk 3 above)
Comment (shoaib812) [reply src/main/java/com/jpn/lang/controller/EmployeeController.java]: I have updated.

**Review by Mohdzaved2434 — APPROVED** [review]

**Pull request merged**